sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/utils.py

```diff
@@ -46,7 +46,7 @@ def dst(self, dt):
 
 
 try:
-    from datetime import timezone  # pylint: disable=ungrouped-imports
+    from datetime import timezone
 
     TZ_UTC = timezone.utc  # type: ignore
```

Comment:
does this help with removing that type ignore?

```suggestion
    TZ_UTC: datetime.timezone = timezone.utc
```

Reply:
ohhh let me try